plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1187,6 +1215,25 @@ def get_diff_merged(self, ap_config):
         self.result["response"] = responses
         return self
 
+    def ap_update_required(self):
+        """
+        Check specified keys are there for AP updation.
+        """
+        required_keys_for_updates = [
+            "ap_name", "admin_status",
+            "led_status", "led_brightness_level", "ap_mode", "location",
+            "failover_priority", "clean_air_si_2.4ghz", "clean_air_si_5ghz", "clean_air_si_6ghz",
+            "primary_controller_name", "primary_ip_address", "secondary_controller_name", "address",
+            "secondary_ip_address", "tertiary_controller_name", "tertiary_ip_address", "2.4ghz_radio",
+            "antenna_name", "radio_role_assignment", "cable_loss", "antenna_cable_name",
+            "channel_assignment_mode", "channel_number", "power_assignment_mode", "powerlevel",
+            "antenna_gain", "channel_width", "5ghz_radio", "6ghz_radio",
+            "xor_radio", "radio_band", "tri_radio", "dual_radio_mode"
+        ]
+        want_key_list = self.want.keys()
+        common_keys = set(want_key_list).intersection(required_keys_for_updates)
```

Comment:
Can we write like below?
`has_required_keys = bool(set(want_key_list).intersection(required_keys_for_updates))`

Reply:
Modified as per your comment